Update support

GraemeConradie opened the pull request:
Fixed newline in contact details.

Zaxounette commented:
@GraemeConradie FYI, your link to https://chat.fintechsystems.ltd/ shows a 502.

Please fix.

GraemeConradie commented:
> @GraemeConradie FYI, your link to https://chat.fintechsystems.ltd/ shows a 502.
> 
> Please fix.

Fixed, moved my chat into a container, apologies.

Zaxounette commented:
Thanks :)
